modules/terraform-aws-vpc/variables.tf

```diff
@@ -183,3 +191,25 @@ variable "vpc_private_outbound_acl_rules" {
     },
   ]
 }
+
+# VPC and Subnet IDs for existing resources (used when create_vpc = false)
+variable "existing_vpc_id" {
+  type        = string
+  description = "ID of existing VPC. Required if create_vpc is false."
```

Comment:
In the end, one more review would be useful to check that variables are grouped based on the two scenarios create\_vpc = true | false.

Reply:
Yes, I've added a note _Only used when create_vpc is true_ to all input variables that are only used when create_vpc is true.